secant/UI Components/Chips/WordChipGrid.swift

```diff
@@ -36,7 +36,7 @@ struct WordChipGrid: View {
 
     init(words: [String], startingAt index: Int, coloredChipColor: Color = .clear) {
```

Comment:
We probably want to apply the same as with Phrase. where we use a `Word` model that can conform to  `Redacted`

I'm mostly thinking of a Handled exception or something of that kind from a crash reporting or event tracking library added to this app or to a fork of it being able to record the seed phrase

Reply:
It's not far fetch'd. It has happened to wallets of other cryptocurrencies

Reply:
updated so words and words: [String] arrays work with RedactableString instead